client/post-editor/edit-post-status/index.jsx

```diff
@@ -1,48 +1,53 @@
 /**
  * External dependencies
  */
-const React = require( 'react' ),
-	noop = require( 'lodash/noop' );
+import React, { PropTypes, Component } from 'react';
+import noop from 'lodash/noop';
+import { moment, translate } from 'i18n-calypso';
```

Comment:
I would recommend not calling on these directly from the component, but instead using the [`localize` higher order component](https://github.com/Automattic/i18n-calypso#localize), which passes both as props to the component.